src/types/custom.ts

```diff
@@ -16,9 +16,11 @@ export interface Fave {
 }
 
 export interface Zoop {
+        receiver: User
+        author: User
```

Comment:
Is this always going to be true? Be careful with this (it will likely be a breaking TS change for the Send Zoop feature, for instance)

Might be worth having a separate ZoopWithDetails object